### PR TITLE
Issue #148

### PR DESCRIPTION
**Pull request by eabovre**
Created 'choices' option in args based on key values in jobconfig.py, also edited 'help' option to match

### CODE REVIEW

File: pds_pipelines/HPCjobber.py [hunk 1]
```diff
@@ -26,11 +26,11 @@ def __init__(self):
         pass
 
     def parse_args(self):
+        choices = jobconfig.keys()
```
Comment (dpmayerUSGS) [inline pds_pipelines/HPCjobber.py]: Let's change this to `choices = list(jobconfig.keys())` so the help statement can just format the list to a string.

Comment (eabovre) [reply pds_pipelines/HPCjobber.py]: Okay, I have updated it, should I close this PR and open a new one?

Comment (dpmayerUSGS) [reply pds_pipelines/HPCjobber.py]: Just commit your edits and push them. The commit will be added to the PR.

Comment (eabovre) [reply pds_pipelines/HPCjobber.py]: Done!

Comment (dpmayerUSGS) [reply pds_pipelines/HPCjobber.py]: Remember to change the help string to just use `.format(choices)` in the help string

**Pull request merged**